### ISSUE TITLE
HTTPS certificate on GitHub pages

### ISSUE THREAD

**Issue by datapythonista** [conda/conda-docs#811]
We're doing some maintenance in the pydata.org domain, and I see that for https://conda.pydata.org/ the certificate was provided via a proxy that it's not needed, and we want to get rid of. Instead of using the provided by github pages in conda.github.io.

Having a look, I'm not sure if the conda.pydata.org is serving what's expected, since https://conda.pydata.org/docs/ seems to be an old version of https://docs.conda.io/en/latest/, which seems to be generated from this same repo. And https://conda.github.io/ doesn't seem to work.

So, I'm wondering if you've got these two different sites, or if there is something causing that old version to be served. If https://conda.pydata.org/ is correct, can you please regenerate its certificate in the GitHub settings, and force https? If you want me to point the conda.pydata.org to the new page, just let me know.

**Comment by jezdez** [comment]
Hey @datapythonista! conda.pydata.org and conda.github.io definitely are outdated that doesn't need to be served anymore. We've just [deleted the gh-pages branch](https://github.com/conda/conda-docs/issues/812) to make sure we're not accidentally leading people to the wrong content.

I think it'd be best to create a URL redirect from conda.pydata.org to docs.conda.io, if that's possible. Can you do that on your end?

**Comment by datapythonista** [comment]
I changed the CNAME of conda.pydata.org to that address. It now fails with an error. I don't know where that page is hosted. Do you want to set up the http server there so it handles the conda.pydata.org domain, or do you prefer that I set up a HTTP permanent redirect in my side?

**Comment by datapythonista** [comment]
Or, maybe if conda.pydata.org wasn't being used and was pointing to an old page, maybe it makes more sense to simply delete it?

**Comment by jezdez** [comment]
Ah, yeah, conda.pydata.org wasn't being used, the main URL for conda's docs is docs.conda.io.

If it's all the same to you, it'd be nice to have a permanent redirect in place from conda.pydata.org to docs.conda.io, so that if there are links anywhere in course materials or slides or something, they would continue to point people to the right site. Would that work?

**Comment by datapythonista** [comment]
Sure, I just implemented the permanent redirect.

It's fine for now, the only small problem I see is that the pydata.org domain has already a huge configuration and growing very quickly. And we're using Cloudflare and have limits on the number of redirects and other things. Happy to leave that redirect just in case for now, but it'll be probably good to eventually remove it, when we're short on the redirects in our plan, or when maintaining the domain becomes complex and slow for all the stuff in there. If any of those really happen. I'd ping you at that point.

Closing this issue as I think we're good for now. Thanks for the help @jezdez, good to work with you again after so many years ;)
